_data-prepper/pipelines/pipelines.md

```diff
@@ -84,269 +88,8 @@ conditional-routing-sample-pipeline:
         hosts: [ "https://opensearch:9200" ]
         index: all_logs
 ```
+{% include copy-curl.html %}
 
+## Next steps
 
-## Examples
-
-This section provides some pipeline examples that you can use to start creating your own pipelines. For more pipeline configurations, select from the following options for each component:
-
-- [Buffers]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/buffers/buffers/)
-- [Processors]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/)
-- [Sinks]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sinks/sinks/)
-- [Sources]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/sources/)
-
-The Data Prepper repository has several [sample applications](https://github.com/opensearch-project/data-prepper/tree/main/examples) to help you get started.
-
-### Log ingestion pipeline
-
-The following example `pipeline.yaml` file with SSL and basic authentication enabled for the `http-source` demonstrates how to use the HTTP Source and Grok Prepper plugins to process unstructured log data:
-
-
-```yaml
-log-pipeline:
-  source:
-    http:
-      ssl_certificate_file: "/full/path/to/certfile.crt"
-      ssl_key_file: "/full/path/to/keyfile.key"
-      authentication:
-        http_basic:
-          username: "myuser"
-          password: "mys3cret"
-  processor:
-    - grok:
-        match:
-          # This will match logs with a "log" key against the COMMONAPACHELOG pattern (ex: { "log": "actual apache log..." } )
-          # You should change this to match what your logs look like. See the grok documenation to get started.
-          log: [ "%{COMMONAPACHELOG}" ]
-  sink:
-    - opensearch:
-        hosts: [ "https://localhost:9200" ]
-        # Change to your credentials
-        username: "admin"
-        password: "admin"
-        # Add a certificate file if you are accessing an OpenSearch cluster with a self-signed certificate  
-        #cert: /path/to/cert
-        # If you are connecting to an Amazon OpenSearch Service domain without
-        # Fine-Grained Access Control, enable these settings. Comment out the
-        # username and password above.
-        #aws_sigv4: true
-        #aws_region: us-east-1
-        # Since we are Grok matching for Apache logs, it makes sense to send them to an OpenSearch index named apache_logs.
-        # You should change this to correspond with how your OpenSearch indexes are set up.
-        index: apache_logs
-```
-
-This example uses weak security. We strongly recommend securing all plugins which open external ports in production environments.
-{: .note}
-
-### Trace analytics pipeline
```

Comment:
Deleted this example because it's redundant to https://opensearch.org/docs/latest/data-prepper/common-use-cases/trace-analytics/#example-trace-analytics-pipeline